tests/eas/cpuhp.py

```diff
@@ -215,3 +215,27 @@ def _random_cpuhp_seq(cls):
         for cpu in cur_off_cpus:
             yield cpu, 1
 
+class ThreeSmallTasks(_CpuHotplugTest):
+    """
+    Test EAS for 3 20% tasks over 60 seconds
+    """
+    workloads = {
+        'cpuhp_three_small' : {
+            'type' : 'rt-app',
+            'conf' : {
+                'class' : 'periodic',
+                'params' : {
+                    'duty_cycle_pct': 20,
+                    'duration_s': 60,
+                    'period_ms': 16,
+                },
+                'tasks' : 3,
+                'prefix' : 'many',
+            },
+        },
+    }
+       
+    @experiment_test
+    def test_random_hotplugs(self, experiment, tasks):
+        return True
```

Comment:
Good point. Looks like `target.check_responsive()` in devlib is a good fit for that